examples/jsm/loaders/KTX2Loader.js

```diff
@@ -270,8 +261,8 @@ class KTX2Loader extends Loader {
 
 	dispose() {
 
-		URL.revokeObjectURL( this.workerSourceURL );
 		this.workerPool.dispose();
+		if ( this.workerSourceURL ) URL.revokeObjectURL( this.workerSourceURL );
```

Comment:
@donmccurdy `dispose()` was implemented two times. I hope this final version is the correct one 😉 .